docs/module_web_with_python.md

```diff
@@ -5,137 +5,437 @@
 ### Clean code
 
 #### Point out 5 suggestions, how to format an SQL query!
+* Separate your query into multiple lines.
+* Capitalize keywords such as "WHERE, HAVING, SELECT" for better readability.
+* Use "AS" to rename columns when needed for a prettier display.
+* Use underscore when naming tables/columns with multiple words in them.
+* Avoid giving the same name for both the table and a column.
 #### What layers can you name in a simple web application?
-
+* Front-end (What happens on the user's computer.)
+* Back-end (What happens on the server's side.)
+* Data storage and access (Usually SQL Datbase, or CSV file.)
 ### Error handling
 #### What error can occur, when an array does not have an element on the requested index?
+* If the requested index is bigger than the length of the array, then an index error occurs.
+* Else, the program runs as it would normally, however, the returned value will be empty.
 #### What is the “finally” block, and how would you use it?
+* The finally block is part of a try-except block.
```

Comment:
Might worth to also mention, as try-catch, as it's more often heard of in programming.

Reply:
fixed